src/EnergyPlus/Autosizing/HeatingWaterDesCoilLoadUsedForUASizing.cc

```diff
@@ -170,6 +171,9 @@ Real64 HeatingWaterDesCoilLoadUsedForUASizer::size(EnergyPlusData &state, Real64
                                            (this->finalSysSizing(this->curSysNum).PreheatTemp - CoilInTemp);
                 }
             } else {
+                if (this->finalSysSizing(this->curSysNum).HeatingCapMethod == DataSizing::FractionOfAutosizedHeatingCapacity) {
+                    this->dataFracOfAutosizedHeatingCapacity = this->finalSysSizing(this->curSysNum).FractionOfAutosizedHeatingCapacity;
+                }
```

Comment:
This is inside the else for if (this->curOASysNum > 0). Couldn't the OA heating coil also be sized using this method? Is so, this conditional should be moved outside this if/else block.

Reply:
This linux build error may show that either this sizing method applies only to air loops, or this method was not properly set up for OA systems. This input is only in the System:Sizing object so that might mean it was only intended for air loop equipment. It would be OK to move this conditional back where it was and we can investigate if OA systems should also allow this same input to adjust sizing for both the air loop coil and OA system coil.

Reply:
That sounds like a plan, I'll move back into the else block and set the OA system question aside for now.